malpolon/tests/test_environmental_raster.py

```diff
@@ -2,11 +2,11 @@
 
 import numpy as np
 import pytest
+import torch
 
 from malpolon.data.environmental_raster import PatchExtractor
 
-
-DATA_PATH = Path("/home/lorieul/dev/research/GeoLifeCLEF22/data")
+DATA_PATH = Path("malpolon/tests/")
```

Comment:
It seems the DATA_PATH is wrong or obsolete and should point to `malpolon.tests.data`
```suggestion
DATA_PATH = Path("malpolon/tests/data")
```
Also, the subdirectory `rasters` is missing so the tests will fail on missing files.

Reply:
ok. Added subdirectory `bioclimatic_rasters` with 3 rasters (18,2MB)